### PR TITLE
Use Mongita by default

### LINKED ISSUE #59: Document about MongoDB / Consider migrating to Mongita
39c569e7c89755e2c9058df618a8df325e5fc080 added pymongo dependency but readme does not say about that 👀

Edit: Please consider [Mongita](https://pypi.org/project/mongita/) btw. MongoDB installation is not very straightforward especially for database newcomers, and Mongita requires no installation. This way you don't have to document anything about MongoDB, and the migration is more than simple as it's (nearly) a drop-in replacement:

```diff
- from pymongo import MongoClient
- mongo_client = MongoClient('mongodb://localhost/')
+ from mongita import MongitaClientDisk
+ mongo_client = MongitaClientDisk()
```

### PR DESCRIPTION
**Pull request by saschanaz**
Closes #59

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ cacheout==0.13.1
 better_profanity==0.6.1
 hatesonar
 pymongo
```
Comment (saschanaz) [inline requirements.txt:7]: Should pymongo still be installed by default if we want mongita as the default one?

File: main.py [hunk 2]
```diff
@@ -28,7 +27,14 @@
 profanity.load_censor_words_from_file('banlist.txt')
 available_songs = get_all_music_available()
 cache = LRUCache()
-mongo_client = MongoClient('mongodb://localhost/')
+
+if os.environ.get("ACE_MONGODB"):
```
Comment (saschanaz) [inline main.py:31]: Hmmm, there is `settings.py` already, maybe it can be used here...

File: requirements.txt (hunk 1 above)
Comment (LuisMayo) [reply requirements.txt:7]: IMO yes, the cost of adding pip dependencies is not really big and allows for easier testing

File: main.py (hunk 2 above)
Comment (LuisMayo) [reply main.py:31]: You're probably right. Consistence is key. You should move it to setting.py and check for None.

Sorry for the trouble

Comment (saschanaz) [reply main.py:31]: Should it be gitignored, btw?

Comment (LuisMayo) [reply main.py:31]: I'm not really sure. Git ignoring it makes it more of a hassle to introduce changes, like this one for instance.

Also I was thinking maybe env variables is better, after all in some PaaS platforms is the only thing you can edit.

So better leave it as it is like env variable. I'll take a look at it when I'm on my PC.
Thanks

**Comment by LuisMayo** [comment]
Merging it! Thanks a lot for the PR!

**Pull request merged**